packages/bot-web-ui/src/pages/bot-builder/bot-builder.tsx

```diff
@@ -100,6 +102,17 @@ const BotBuilder = observer(() => {
             },
         });
     };
+    useEffect(() => {
+        setTimeout(() => {
+            if (isSafari()) {
+                const images: NodeListOf<SVGImageElement> = document.querySelectorAll('.blocklyEditableText image');
+                images.forEach((img: SVGImageElement) => {
```

Comment:
Also instead of the foreach loop can we write a custom function and call in the dropdown files where it is needed